app/views/shared/_nav_auth.html.slim

```diff
@@ -12,7 +12,10 @@ nav.bg-white
           = link_to t('shared.nav_auth.news'), root_path, class: 'mr3 gray'
           = link_to t('shared.nav_auth.contact'), root_path, class: 'mr3 gray'
       .sm-col-right.sm-right-align
-        div = t('shared.nav_auth.welcome', email: current_user.email)
+        div
+          <strong>
```

Comment:
they were both bolded before: https://github.com/18F/identity-idp/commit/44349df995f51897f37f9688384476dedf64c6f4#commitcomment-19664904

but I can update if you prefer us to change how it was?